src/components/drivers/photo-viewer-wrapper.jsx

```diff
@@ -29,8 +29,15 @@ export default class PhotoViewerWrapper extends Component {
     this.loadFile();
   }
 
-  componentDidUpdate() {
-    this.loadFile();
+	componentDidUpdate(prevProps, prevState, prevContext) {
```

Comment:
since we don't use the undocumented `context` param, probably better to leave it off